juniper_official/lldp/lldp-session-state.rule

```diff
@@ -67,13 +62,12 @@ healthbot {
                  */
                 term is-lldp-session-up {
                     when {
-                        matches-with "$neighbor-state" true;
-                        matches-with "$if-lldp-state" true;
+                        matches-with "$interface-lldp-state" true;
                     }
                     then {
                         status {
                             color green;
-                            message "Interface $interface-name lldp session is up";
+                            message "LLDP interface $interface-name status is Up";
```

Comment:
Change this to "LLDP status on interface $interface-name is UP".
LLDP status is coming as true and we are displaying message as UP. Is this correct? Should we say that LLDP on interface is enabled or something like that?

Reply:
Also line 81 `message "Interface-$interface-name lldp session down";`
Please put a space after "Interface" and remove "-"

Reply:
updated as per the comments

Reply:
You did not answer this questions "LLDP status is coming as true and we are displaying message as UP. Is this correct? Should we say that LLDP on interface is enabled or something like that?"

Please check with Venkat and comment if this is ok or not